resources/conf/config.toml.template

```diff
@@ -492,6 +492,10 @@ enabled = true
   # Enforcer related configurations
   [analytics.enforcer]
     [analytics.enforcer.configProperties]
+      # Enables elk analytics logging
```

Comment:
Depending on the comment below, the comment could be more descriptive saying that if the field remains unassigned it would be choreo analytics using.